### ISSUE TITLE
Browser for Strip

### ISSUE THREAD

**Issue by vcvrackideas** [stoermelder/vcvrack-packone#108]
One thing that still take me apart from permanent use of Strip as collection of "effect racks" is a lack of kind of internal mini browser even in right click menu. Could you say is it hard to implement some default user library folder and cache strip files to a strip for faster loading? The best looking and ux thing I've ever seen is Abletons user library, you can just drop any new effect or synth rack to it, then name it and then you can recall each of saved racks just by double clicking. So in that way naming inside of Strip while saving would be also neat addition. But I really don't know if VCV allows that kind of non explorer\non finder operations...

**Comment by stoermelder** [comment]
There are quite some limitiations in the user interface of Rack. While investigating an idea today I found out that it's possible to create a custom overlay like the module browser. It would take some amount of work but something similar for presets would be possible...

**Comment by vcvrackideas** [comment]
Wow, looking forward for any solution you will find!

**Comment by stoermelder** [comment]
Any suggestion how this could look like? A mockup maybe?

**Comment by vcvrackideas** [comment]
Not really, I was only thinking about the list of presets and subgroups in right click menu, but I can't imagine yet how's custom overlay can be, could it contains some screenshot of strip? So technically is it possible to get some overlay like module browser on right click? If you can explain the technical limits of this I will think about mockup for sure!

**Comment by VegaDeftwing** [comment]
Hmm, I don't imagine a screenshot would be easily doable, so maybe even just a better load menu that opened a 'window' internal to rack instead of relying on the file manager that lists what modules are in the strip file? Also, I could see strip being expanded to include some in and out ports (basically an internal send/return, Like Vult's 'Send', but no knob) that are saved in the strip's wiring so that multiple strips could be  tested rapidly, as the strip module's own connections wouldn't have to be reset on each load. This would obliviously necessitate the modules being larger, but could be very handy.

**Comment by stoermelder** [comment]
This is a very old request, but a browser for vcv/vcvs/vcvss-files is finally coming to my MB module.

**Comment by stoermelder** [comment]
https://github.com/stoermelder/vcvrack-packone/issues/436
